install.sh

```diff
@@ -521,8 +521,8 @@ sudo pip install -q -U tqdm
 # The BlueView SDK for the Teledyne imaging sonar
 if [[ ! -z "$BVTSDK_PASSWORD" ]]; then
 	instlog "Decrypting and installing the BlueView SDK"
-	cd $CATKIN_DIR/src
-	curl -s https://raw.githubusercontent.com/uf-mil/installer/master/bvtsdk.tar.gz.enc | \
+	cd $CATKIN_DIR/src/mil_common/drivers/mil_blueview_driver/
+	curl -s https://raw.githubusercontent.com/uf-mil/installer/master/libbvtsdk.tar.gz.enc | \
 	openssl enc -aes-256-cbc -d -pass file:<(echo -n $BVTSDK_PASSWORD) | tar -xpz
 	if [[ ! -d $CATKIN_DIR/src/bvtsdk ]]; then
```

Comment:
Why did you not change this? Be sure you look over everything relating to the SDK.